Mathlib/Algebra/Order/Group/Unbundled/Int.lean

```diff
@@ -129,6 +132,10 @@ protected theorem sign_eq_ediv_abs (a : ℤ) : sign a = a / |a| :=
   if az : a = 0 then by simp [az]
   else (Int.ediv_eq_of_eq_mul_left (mt abs_eq_zero.1 az) (sign_mul_abs _).symm).symm
 
+protected theorem sign_eq_abs_ediv (a : ℤ) : sign a = |a| / a :=
```

Comment:
I think this one could be a good simp lemma backwards?

Reply:
Perhaps, but this matches the one above it, so that sounds like a follow-up